docs/usage.md

```diff
@@ -1,6 +1,71 @@
+
+- **Streamline**: For tasks related to setting up the incident response platform, which is augmented with automation of common tasks such as enrichment and mitigation.
+
+- **Accelerate**: For tasks related to accelerating your response time by leveraging various pre-defined playbooks to respond to specific types of threats, and installing solution packs targeted towards management of specific SOAR scenarios, such as Threat Intel management, Vulnerability Management, etc.
+
+- **Maintain**: For tasks related to enabling monitoring and sustenance so that the server performs optimally and remains highly available throughout your usage.
+
+FortiSOAR Setup Guide opens in the Quick Start mode that contains limited items to set up. To view the complete list of tasks:
+
+1. Select **Production**.
+
+2. Click '![right icon](./res/icon-chevron-right.svg)' against each section to expand all its tasks. The following image shows the expanded task *Leveraging the FortiSOAR Trial License for Development Environments* under Deploy.
+
+    ![Setup Guide - Production tasks and details for the Deploy task](./res/setup-guide-expanded.png)
+
+Each task contains a brief description and a link to either the documentation containing detailed information, or, the FortiSOAR page or section to set up that particular task. For example:
+
+- Click the **sizing guide** link in **Review the Sizing and Configuration** row to open the *Sizing Guide* and check the configuration for your anticipated workload. 
+
+- Click **Review** in the **Define Notification Rules** task to open the **Notifications** page. Here, you can set up rules and notifications channels to notify users and teams various tasks they need to complete or view.
+
+For each task you have three options:
+
+- **Mark as Done**: Click this button to mark the task as done, once you complete it.
+
+    ![](./res/setup-guide-done.png)
+
+    - Click **Mark as Done** to display that task with a green circular check ![icon-green-check](./res/icon-mark-as-done.svg). A **Reset Task Options** button appears in its place.
+
+    - Click **Reset Task Options** button to bring back the options, in case you clicked **Marked as Done** in error.
+
+- **I will complete later**: Select this button to skip that task and complete it at a later time.
+
+    - Click **I will complete later** to collapse that task's row
+
+    - The task action buttons remain active and appear when the row is expanded
+
+- **Not Applicable**: Select this button if a task does not apply to your FortiSOAR environment.
+
+    ![](./res/setup-guide-na.png)
+
+    - Click **Not Applicable** to display that task with a struck-out, circular check ![icon-green-check](./res/icon-not-applicable.svg). A **Reset Task Options** button appears in its place.
```

Comment:
"icon-not-applicable" should be shown instead of "icon-green-check"

Reply:
You mean the `alt=` text. Will change. Thanks. Check in dark mode as well.